earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/build_and_test.py

```diff
@@ -91,6 +99,8 @@ def BuildLibrary(os_dir, ignore_results):
     archData = platform.architecture(pathToLib)
     if archData[0] == "64bit":
       specialDefs = "-DMS_WIN64"
+  elif os_dir == "Linux":
+    build_swig(source_dir)
```

Comment:
We only need to do this on Linux?  What about Mac or Windows?

Reply:
I honestly don't know about mac, but windows we have the newer version of swig available to us, so we don't need to build it.

Reply:
Looks like according to docs Portable is not supported on Mac so we can forget about that.  I do thing the docs need to be updated to reflect these changes: https://www.opengee.org/geedocs/5.3.4/answer/portable/portableUserGuideWinLinux.html

Reply:
Ok. That seems logical to me. Ill make reviewing the changes to the wiki part of reviewing the PR.